donjo-app/models/Analisis_kategori_model.php

```diff
@@ -81,17 +81,19 @@ public function list_data($o=0, $offset=0, $limit=500)
 
 	public function insert()
 	{
-		$data = $_POST;
-		$data['id_master'] = $_SESSION['analisis_master'];
+		$data = [];
+		$data['id_master'] = $this->session->analisis_master;
+		$data['kategori'] = htmlentities($this->input->post('kategori'));
 		$outp = $this->db->insert('analisis_kategori_indikator', $data);
 
 		status_sukses($outp); //Tampilkan Pesan
 	}
 
 	public function update($id=0)
 	{
-		$data = $_POST;
-		$data['id_master']=$_SESSION['analisis_master'];
+		$data = [];
+		$data['id_master']=$this->session->analisis_master;
```

Comment:
Perbaiki penulisan mengikuti aturan `$data['id_master'] = $this->session->analisis_master;`